### PR TITLE
[JENKINS-35323] Migrate amazon-ecr-plugin to 2.x parent pom

### PR DESCRIPTION
**Pull request by fbelzunc**
@reviewbybees CC @andresrc @ndeloof

### CODE REVIEW

**Comment by fbelzunc** [comment]
@reviewbybees done

Closing - there are some problems

File: pom.xml [hunk 1]
```diff
@@ -29,9 +29,9 @@
   <modelVersion>4.0.0</modelVersion>
 
   <parent>
-    <groupId>com.cloudbees.jenkins.plugins</groupId>
-    <artifactId>jenkins-plugins</artifactId>
-    <version>39</version>
+    <groupId>org.jenkins-ci.plugins</groupId>
+    <artifactId>plugin</artifactId>
+    <version>2.7</version>
```
Comment (andresrc) [inline pom.xml]: Go to 2.9

File: pom.xml [hunk 2]
```diff
@@ -45,6 +45,8 @@
 
   <properties>
     <jenkins.version>1.609</jenkins.version>
+    <java.level>7</java.level>
```
Comment (andresrc) [inline pom.xml]: 1.609 is Java 6. Any reason to need Java 7?

File: pom.xml [hunk 3]
```diff
@@ -60,6 +62,18 @@
...
+        <configuration>
+          <excludeFilterFile>findbugs-exclude.xml</excludeFilterFile>
+        </configuration>
+      </plugin>
+    </plugins>
+  </build>
+
   <repositories>
     <repository>
       <id>repo.jenkins-ci.org</id>
```
Comment (andresrc) [inline pom.xml]: Move the repos to `https`

**Pull request closed without merging**